shared/uavobjectdefinition/stabilizationsettings.xml

```diff
@@ -27,6 +27,9 @@
 		<field name="PitchPI" units="" type="float" elementnames="Kp,Ki,ILimit" defaultvalue="2.5,0,50" limits="%BE:0:20,%BE:0:20,"/>
 		<field name="YawPI" units="" type="float" elementnames="Kp,Ki,ILimit" defaultvalue="2.5,0,50" limits="%BE:0:20,%BE:0:20,"/>
 		<field name="GyroCutoff" units="Hz" type="float" elements="1" defaultvalue="55.0"/>
+		<field name="GyroButterworth" units="" type="enum" elements="1" options="FALSE,TRUE" defaultvalue="FALSE">
+			<description>Enables the use of a second order Butterworth lowpass filter on the gyro. Can't be changed during runtime.</description>
```

Comment:
It would be really nice if it could--- isn't this just a question of moving the malloc of the butterworth filter down, like..

```
  else {
    if (!butter_gyro_filter) {
      butter_gyro_filter = PIOS_malloc_no_dma(sizeof(butterworth_filter);
    }
    if (butter_gyro_filter) {
      create_butterworth(butter_gyro_filter, settings.GyroCutoff, dT_expected);
    }
  |